playbooks/testdrive.yml

```diff
@@ -36,12 +36,13 @@
     - edxlocal
     - role: mongo
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
+    # Do we relly need edxapp 2x times here?
     - { role: 'edxapp', celery_worker: True }
     - edxapp
-    - notifier
     - analytics_api
     - insights
     - edx_notes_api
+    # Do we really need this 2x times?
```

Comment:
I think so because the first one imports the default demo course, and second one imports a different course. Yes, the role can be rewritten to allow passing of a list of courses instead of running it multiple times, but again, not sure we want to spend that time considering it's testdrive.